lib/service/InitService.ts

```diff
@@ -75,12 +82,26 @@ class InitService extends EventEmitter {
     }
   }
 
-  public restoreNode = async (args: { password: string, seedMnemonicList: string[] }) => {
-    const { password, seedMnemonicList } = args;
+  public restoreNode = async (args: RestoreNodeRequest.AsObject) => {
```

Comment:
Although I can see the benefit/convenience of using the `AsObject` type here, we have been keeping the grpc types separate from the service/initservice classes. In this case though I think there would actually be some benefit to explicitly defining the parameter types for this method if we switch from `string` to `bytes` for the protobuf definition. The generated type for `bytes` is `string | Uint8Array` however that's only so it can accept strings when setting parameters. If we are getting a `bytes` value it will always be `Uint8Array` and we could explicitly define that here, then tweak the GrpcInitService.ts class to pass in all `bytes` parameters `as Uint8Array` to this method and have stricter types going forward.